BeVolt/App/src/Temperature.c

```diff
@@ -4,12 +4,26 @@
  */
 
 // NOTE: All units are in Celsius
+// TODO: Create more testing cases
+// TODO: Make sure data type is consistent, need to revise if going to Fixed Point w/ .01 resolution
+// TODO: Maybe make temperature boards more clean later on once system is more finalized.
+// TODO: How to reset danger flags?
 
 #include "Temperature.h"
 #include "config.h"
 
-uint16_t ModuleTemperatures[NUM_TEMPERATURE_BOARDS][20];	// Each board may not have 20 sensors. Look at config.h for how many sensors
+int16_t ModuleTemperatures[NUM_TEMPERATURE_BOARDS][20];	// Each board may not have 20 sensors. Look at config.h for how many sensors
 																												// there are on each board. 20 is just max that the board can handle
+int16_t ModuleTempStatus[NUM_TEMPERATURE_BOARDS];
+
+typedef enum chargeState {
+	discharging,
+	charging
+} chargeState;
```

Comment:
This is never used. It should either be deleted, or the variable `chargingState` should be changed to this type in order to put it to good use. 

Changing the variable's type and the associated function arguments would probably make the most sense, since there is a very limited number of states that the `chargingState` is restricted to.

```suggestion
} ChargeState;
```